server/src/utils/parseJwt.ts

```diff
@@ -0,0 +1,5 @@
+import * as jwt from 'jsonwebtoken'
+
+export function parseJwt(token: string) {
+  return jwt.verify(token, 'lavender_snake_secret_key')
```

Comment:
In real life, this should never be checked into github, even temporarily :P Normally there's some sort of external secrets management tool (I'm using https://cloud.google.com/secret-manager since I'm hosting on GCP), where secrets are made available via the environment, so you'd access something like `process.env.JWT_PRIVATE_KEY`

Reply:
I'll go straight with env next time